index.bs

```diff
@@ -9777,6 +9788,15 @@ The [=remote end steps=] given |session| and |command parameters| are:
 1. Let |arguments| be the <code>arguments</code> field of |command
    parameters| if present, or an empty [=/list=] otherwise.
 
+1. Let |user contexts| be [=set/create=] with |command parameters|[<code>userContexts</code>]
+   if |command parameters| [=map/contains=] [<code>userContexts</code>] and an empty [=/set=] otherwise.
```

Comment:
Square brackets not necessary. Again, sometimes we seem to use quotes, sometimes not. Feel free to add or not
```suggestion
   if |command parameters| [=map/contains=] <code>userContexts</code> and an empty [=/set=] otherwise.
```